### PR TITLE
Fix/header base card

### PR DESCRIPTION
**Pull request by R-Jim**
Header now will have different background based on media width

### CODE REVIEW

File: frontend/src/shared/components/baseCard/baseCard.css [hunk 1]
```diff
@@ -48,20 +46,16 @@
 
 .card-content {
     --card-text-width: 420px;
-
     display: block;
     width: 100%;
-    min-height: 1px;  /*DO NOT REMOVE THIS LINE. OTHERWISE YOU WILL EXPERIENCE PAIN IN TRYING TO FIX COLUMN STACKING ON CHROME/IOS*/
+    min-height: 1px;
+    /*DO NOT REMOVE THIS LINE. OTHERWISE YOU WILL EXPERIENCE PAIN IN TRYING TO FIX COLUMN STACKING ON CHROME/IOS*/
```
Comment (zapplecat) [inline frontend/src/shared/components/baseCard/baseCard.css]: This is now ambiguous what line not to remove

**Review by zapplecat — APPROVED** [review]
lgtm other than the small comment on the comment.

File: frontend/src/shared/components/baseCard/baseCard.css (hunk 1 above)
Comment (R-Jim) [reply frontend/src/shared/components/baseCard/baseCard.css]: somehow the code formatter decide to change this one. I will revert it

Comment (DuongTyler) [reply frontend/src/shared/components/baseCard/baseCard.css]: I should make that comment more descriptive sometime :P

**Review by DuongTyler — APPROVED** [review]
looks good

**Pull request merged**